datasets/finance.py

```diff
@@ -29,4 +29,4 @@ def get_data(self):
 
         data = dict(X=self.X, y=self.y)
 
-        return data
+        return data
```

Comment:
there should be a newline at then end of file, you can configure your editor to add it on save